### ISSUE TITLE
Component: Micro Footer - Color contrast on Languages for example:- German, English etc

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#3816]
## Detailed description

Issue: Foreground:#A8A8A8
Background:#FFFFFF

The contrast ratio is: 2.4:1
Text failed at Level AA

Expected:
Provide below contrast ratio.
Foreground:#6D6D6D
Background:#FFFFFF

The contrast ratio is:5.2:1
Text passed at Level AA

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/c228789d-ad22-4b8e-80db-a4045c5aaeef)

**Comment by kennylam** [comment]
Duplicate of https://github.com/carbon-design-system/ibm-dotcom-library/issues/3445
